core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -1736,7 +1736,7 @@ public function getArticles(DataContainer $dc)
 			while ($objArticle->next())
 			{
 				$key = $objArticle->parent . ' (ID ' . $objArticle->pid . ')';
-				$arrArticle[$key][$objArticle->id] = $objArticle->title . ' (' . ($GLOBALS['TL_LANG']['COLS'][$objArticle->inColumn] ?: $objArticle->inColumn) . ', ID ' . $objArticle->id . ')';
```

Comment:
The same code is also in line 1416 and in `tl_news.php:811`. Why did you only fix it here? Plus there are two occurrences of `] ?: $` in `DC_Table` and `DC_Folder`, which we should check as well.

Reply:
I only got a warning at the point

Reply:
I cannot merge the PR unless all occurrences of the problem have been fixed. 🤷‍♂️